src/getCoinCombination.test.js

```diff
@@ -7,4 +7,20 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('should return coins if a values([1, 1, 0, 0])', () => {
+    expect(getCoinCombination(6));
```

Comment:
```suggestion
    expect(getCoinCombination(6)).toEqual([1, 1, 0, 0]);
```
Your solution will not work. You need to make changes according to my example